draft-yasskin-wpack-bundled-exchanges.md

```diff
@@ -268,6 +268,11 @@ steps, taking the `stream` as input.
    `knownSections` says not to process other sections, add those sections' names
    to `ignoredSections`.
 
+   Note: The `ignoredSections` enables sections that supercedes other sections
+   to be introduced in the future. Since there are no such sections in the
```

Comment:
How about

> Implementations that don't implement any such sections are free to omit the relevant steps.

since even after we add such a section to the registry, implementations that don't implement it will still be able to skip the `ignoredSections` steps.

I expect to get some pushback based on the fact that implementations are [always allowed to differ from the specified steps as long as they get the same answer](https://infra.spec.whatwg.org/#algorithms).